src/lib/certificate-manager.ts

```diff
@@ -148,11 +152,65 @@ export async function openCertificate() {
 	shell.showItemInFolder( CA_CERT_PATH );
 }
 
+/**
+ * Checks if the root CA certificate is already trusted by the system
+ * @returns A promise that resolves to true if the certificate is trusted, false otherwise
+ */
+export async function isRootCATrusted(): Promise< boolean > {
+	if ( ! fs.existsSync( CA_CERT_PATH ) ) {
+		return false;
+	}
+
+	if ( process.platform === 'win32' ) {
+		try {
+			// Execute certutil with more specific validation
+			const { stdout } = await execFilePromise( 'certutil', [
+				'-verify',
+				'-urlfetch',
```

Comment:
```suggestion
```
Let's remove this option. The `certutil` help docs say this about the `-verify` option:

> Retrieve and verify AIA Certs and CDP CRLs

The way I understand it, it means that `certutil` will potentially check the validity of the trust chain by downloading data from Microsoft servers. Since our CA is generated locally, it's not trusted by any global CAs, and we can skip this procedure altogether.

Reply:
Removed `-urlfetch` parameter.